### PR TITLE
chore(deps): update paragon and frontend-build to openedx scope

### PR DESCRIPTION
**Pull request by brian-smith-tcril**
## This is blocking #846 which also needs to get merged ASAP

Includes all changes from https://github.com/openedx/frontend-app-payment/pull/838, but also updates `frontend-component-header`, `frontend-component-footer`, and `frontend-platform` to versions with updated peer dependencies that use the `@openedx` scope.

### CODE REVIEW

**Comment by brian-smith-tcril** [comment]
@zubair-ce07 I see you approved https://github.com/openedx/frontend-app-payment/pull/846, however, it is currently failing CI due to allowlist issues. This PR resolves those issues. If you could review/merge this PR so that one can be rebased that would be incredibly helpful. Thanks!

**Comment by brian-smith-tcril** [comment]
@openedx/revenue-squad is anyone available to review/merge this? It resolves the allowlist audit problem that is causing https://github.com/openedx/frontend-app-payment/pull/846 to fail CI.

File: package.json [hunk 1]
```diff
@@ -31,15 +31,15 @@
   },
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
-    "@edx/frontend-component-footer": "12.2.0",
-    "@edx/frontend-component-header": "4.6.0",
-    "@edx/frontend-platform": "5.5.4",
-    "@edx/paragon": "^20.44.0",
+    "@edx/frontend-component-footer": "13.0.2",
+    "@edx/frontend-component-header": "5.0.2",
+    "@edx/frontend-platform": "7.1.0",
```
Comment (OmarIthawi) [inline package.json:36]: Are these imports supposed to stay `@edx`?

**Review by OmarIthawi — APPROVED** [review]
Thanks @brian-smith-tcril ! I've checked the code. I have one question, but it looks good to me.

File: package.json (hunk 1 above)
Comment (brian-smith-tcril) [reply package.json:36]: Yes. As of writing, all 3 are still being published to `@edx`

https://github.com/openedx/frontend-component-footer/blob/62bf038a06feac7865e558205b29bc726e9594c4/package.json#L2
https://github.com/openedx/frontend-component-header/blob/3b2a2bfa9576926fcf6278b222835d68752bba0e/package.json#L2
https://github.com/openedx/frontend-platform/blob/f0e120244bb367b524c72ba30ef7e09c6b2cbcfc/package.json#L2

Comment (OmarIthawi) [reply package.json:36]: Thanks @brian-smith-tcril. Any chance we could merge this pull request?

cc: @zubair-ce07

**Comment by OmarIthawi** [comment]
@brian-smith-tcril Would it be acceptable if I cherry-pick this commit, test it locally on my machine and include it into https://github.com/openedx/frontend-app-payment/pull/846

It's the only remaining PR of ~15 other PRs that has been merged.

cc: @revenue-squad @zubair-ce07 @sarina

File: package.json (hunk 1 above)
Comment (zubair-ce07) [reply package.json:36]: On it.

**Review by zubair-ce07 — APPROVED** [review]

**Comment by OmarIthawi** [comment]
Thanks @zubair-ce07!!

**Comment by sarina** [comment]
@zubair-ce07 - only people on the Revenue squad can merge PRs to this repo. Could you merge ASAP?

**Comment by zubair-ce07** [comment]
> @brian-smith-tcril Would it be acceptable if I cherry-pick this commit, test it locally on my machine and include it into #846
> 
> It's the only remaining PR of ~15 other PRs that has been merged.
> 
> cc: @revenue-squad @zubair-ce07 @sarina

Yes sure 👍

**Pull request merged**